Readme.md

```diff
@@ -26,7 +26,7 @@ err:= client.ExecuteInto(request, &m)
 
 // or
```

Comment:
Can we add comments like `Fixed` or `done` or raise any concern that we may have with the review comments. It will help speed up the review process?

Reply:
Added the Fixed comments and I also resolved them. Let me know what if that isn't what I was supposed to do.